cmd/srcd/cmd/parse.go

```diff
@@ -133,18 +157,13 @@ var parseLangCmd = &cobra.Command{
 		if err != nil {
 			return fmt.Errorf("could not get daemon client: %v", err)
 		}
-		ctx, cancel := context.WithTimeout(context.Background(), time.Second)
-		defer cancel()
 
-		res, err := c.Parse(ctx, &api.ParseRequest{
-			Kind:    api.ParseRequest_LANG,
-			Name:    path,
-			Content: b,
-		})
+		lang, err := parseLang(context.Background(), c, path, b)
 		if err != nil {
-			return fmt.Errorf("server error: %v", err)
+			return fmt.Errorf("cannot parse language: %v", err)
```

Comment:
huh? You parse a file here.

Reply:
What I meant is that it cannot parse the language from the file. Do you think that `"cannot parse language from given file: %v"` or `"cannot parse file: %v"` is better?

Reply:
ah, wait. It's my bad. The context was collapsed on GH and I misunderstood it. All fine. Thanks!